### DISCUSSION TITLE
Request is yielding forever

### DISCUSSION THREAD

**Discussion by jmkd3v** [encode/httpx#1654] [Potential Issue]
`httpx.AsyncClient.request` is yielding forever. After debugging for 20 minutes and trying to find the function that was yielding, I've narrowed down the issue to this code in _client.py:
```py
with request_context(request=request):
    (
        status_code,
        headers,
        stream,
        extensions,
    ) = await transport.handle_async_request(
        request.method.encode(),
        request.url.raw,
        headers=request.headers.raw,
        stream=request.stream,
        extensions={"timeout": timeout.as_dict()},
    )
```
I checked the source of `handle_async_request` and it is as follows:
```py
raise NotImplementedError(
    "The 'handle_async_request' method must be implemented."
)  # pragma: nocover
```
This is incredibly frustrating as sometimes it just starts happening when I drop in another library (in this case, Flask 2 with async) and completely breaks our workflow.

**Comment by jmkd3v** [comment]
Small update. This seems to only occur when I start the function with `asyncio.create_task` - just a plain old await works fine.

**Reply by jmkd3v** [reply]
Moving over to Quart seems to fix this issue.

**Reply by lovelydinosaur** [reply]
You'd need to give a clearer reproducible example in order for someone to be able to give you a hand with this.

**Reply by jmkd3v** [reply]
I honestly have no idea why it does this, so it's hard for me to make a reproducible example.
